src_analysis/benchmark_dependencies.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Google LLC
```

Comment:
In my mind it makes sense to call this directory src_analysis instead of src_analyzer because it isn't an independent module used in experiments like "runner" or "measurer". What do people think?